ci/pipelines/toolbox-staging-ecr.yml

```diff
@@ -0,0 +1,165 @@
+    source:
+      repository: govukpay/toolbox
+      aws_access_key_id: ((readonly_access_key_id))
+      aws_secret_access_key: ((readonly_secret_access_key))
+      aws_session_token: ((readonly_session_token))
+      # Need to create the role for the concourse user to assume in the test account
+      aws_role_arn: arn:aws:iam::((pay_aws_test_account_id)):role/concourse
+      # Hardcode the test account registry ID for now. Needs to be a string, not a number
+      aws_ecr_registry_id: "((pay_aws_test_account_id))"
+      aws_region: eu-west-1
+      tag: ready-staging
+  # ECR Staging registry resource
+  - name: ecr-registry-staging
+    type: dev-registry-image
+    icon: docker
+    source:
+      repository: govukpay/toolbox
+      aws_access_key_id: ((readonly_access_key_id))
+      aws_secret_access_key: ((readonly_secret_access_key))
+      aws_session_token: ((readonly_session_token))
+      # Need to create the role for the concourse user to assume in the staging account
+      aws_role_arn: arn:aws:iam::((pay_aws_staging_account_id)):role/concourse
+      # Hardcode the staging account registry ID for now. Needs to be a string, not a number
+      aws_ecr_registry_id: "((pay_aws_staging_account_id))"
+      aws_region: eu-west-1
+
+resource_types:
+  # Custom resource type - this has been merged to the
+  # https://github.com/concourse/registry-image-resource repo,
+  # however is not yet in a stable release. Using this custom
+  # resource type allows us to support the ECR registry_id parameter
+  # This can be removed once registry_id is in stable release of 'registry-image'.
+  - name: dev-registry-image
+    type: registry-image
+    source:
+      repository: concourse/registry-image-resource
+      username: ((docker-username))
+      password: ((docker-password))
+      tag: dev
+
+jobs:
+  - name: update-toolbox-staging-ecr-pipeline
+    plan:
+      - get: toolbox-staging-ecr
+        trigger: true
+      - set_pipeline: toolbox-staging-ecr
+        file: toolbox-staging-ecr/ci/pipelines/toolbox-staging-ecr.yml
+  - name: toolbox-staging-approval
+    plan:
+      - get: ecr-registry-test
+        trigger: true
+        params:
+          format: oci
+      - task: pull-tags-from-ecr-registry-test
+        config:
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: amazon/aws-cli
+          params:
+            AWS_ACCESS_KEY_ID: ((readonly_access_key_id))
+            AWS_SECRET_ACCESS_KEY: ((readonly_secret_access_key))
+            AWS_SESSION_TOKEN: ((readonly_session_token))
+            AWS_ASSUME_ROLE_ARN: arn:aws:iam::((pay_aws_test_account_id)):role/concourse
+            AWS_REGISTRY_ID: "((pay_aws_test_account_id))"
+            AWS_REGION: eu-west-1
+            AWS_PAGER: ""
+            AWS_DEFAULT_OUTPUT: text
+          inputs:
+            - name: ecr-registry-test
+          outputs:
+            - name: docker_tags
```

Comment:
I think the term `image_tags` makes more sense to me than `docker_tags` but others might disagree.

Reply:
Happy to stick with `docker_tags` for now, given we're using it in the test pipeline as well